lib/airbrake-ruby/config.rb

```diff
@@ -167,6 +176,15 @@ def logger=(logger)
       @logger = logger || @logger
     end
 
+    # Sets the default method wrapping style. Must be :chain or :prepend.
+    def default_wrapping_style=(style)
+      style = style.to_sym if style.respond_to?('to_sym')
```

Comment:
Why do we need to check if `style` responds to `to_sym`?
The expected argument is a String or a Symbol. They both seem to support `to_sym` since ancient times.
If we pass any other object, it'll fail with a `NoMethodError` and that's okay.

Reply:
I prefer to raise better errors. Unhandled NoMethodError is almost always an indication of a bug or a typo in your code, so if you have your code raise NoMethodError as part of your official API then your tests won't be able to tell the difference between that NoMethodError and one that was raised because you mistyped something in your code.

Reply:
I verified this in my tests. I added the tests and checked for ArgumentError, but didn't make the comma change below. It raised a NoMethodError instead, indicating a bug. If I had been expecting a NoMethodError I wouldn't have caught the bug. It's almost never good practice to raise NoMethodError on purpose.

Reply:
FWIW we wouldn't raise a `NoMethodError` on purpose. We would just not perform any "typechecking". That said, what you said sounds reasonable to me, so no objections (especially since this will be called only once, on process initialisation).